README.md

```diff
@@ -1,2 +1,63 @@
 # Mergebot
-Mergebot is a GitHub Action that automates merging Dependabot PRs
+
+Mergebot is a GitHub Action that automates merging Dependabot PRs. It supports semantic versioning, enabling you to choose how big of an upgrade bump you allow, and automatically adds reviewers in case the PR requires special attention
+
+Even though it is designed with Dependabot in mind, Mergebot can also be used to automate other GitHub users' PRs. You simply need to override the default input for this
+
+# Dependabot config
+
+Below is a sample dependabot.yml. Notice the schedule is set to daily. This makes the process less conflict-prone. Mergebot can run with any Dependabot config
+
+    version: 2
+    updates:
+      - package-ecosystem: docker
+        directory: /
+        schedule:
+          interval: daily
+      - package-ecosystem: npm
+        directory: /
+        schedule:
+          interval: daily
+      - package-ecosystem: nuget
+        directory: /
+        schedule:
+          interval: daily
+
+# Mergebot config
+
+Below is a sample workflow that uses Mergebot. Notice we recommend checking whether the check_suite was triggered by Dependabot and it finished successfully. This is not necessary, as the action will check this also, but it saves a few seconds from the runtime
```

Comment:
If this is not necessary, and it saves a few seconds only, I recommend not including it in the sample config.

Reply:
Sure